src/include/platform/ThreadStackManager.h

```diff
@@ -97,6 +97,7 @@ class ThreadStackManager
     CHIP_ERROR GetAndLogThreadTopologyMinimal();
     CHIP_ERROR GetAndLogThreadTopologyFull();
     CHIP_ERROR GetPrimary802154MACAddress(uint8_t * buf);
+    CHIP_ERROR GetIeeeEui64(uint8_t * buf);
```

Comment:
Please document (e.g. how big the buffer needs to be).

Reply:
Of course, thank you for pointing this out!

Reply:
Updated here: https://github.com/project-chip/connectedhomeip/pull/15367/commits/8c702c8ab0a735f76a79de460b268a91687d4cf4